procurement_purchase_requisition_generation/models/stock_rule.py

```diff
@@ -0,0 +1,85 @@
+# Copyright 2025 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from collections import defaultdict
+
+from odoo import SUPERUSER_ID, api, models
+
+
+class StockRule(models.Model):
+    _inherit = "stock.rule"
+
+    def _make_pr_get_domain(self, company_id, values):
```

Comment:
Please use other function name to avoid conflict with `purchase_request` module
https://github.com/OCA/purchase-workflow/blob/1ff6fee1645b9fc42596d2509f9f0268c54e9529/purchase_request/models/stock_rule.py#L49

```suggestion
    def _make_purchase_requisition_domain(self, company_id, values):
```
![image](https://github.com/user-attachments/assets/296d6315-4cdb-4c8a-9b5b-b4ce01263f32)